pkg/rancher-desktop/backend/lima.ts

```diff
@@ -96,7 +101,7 @@ export type LimaConfiguration = {
   memory?: number;
   disk?: number;
   mounts?: LimaMount[];
-  mountType: string;
+  mountType: 'reverse-sshfs' | '9p' | 'virtiofs';
```

Comment:
Yeah, not sure. I haven't done it because the enums are for our own Settings structure, not for the LimaConfig. The values have just been chosen so that we can simply copy them over.

I'll leave it separate for now, but may change my mind later.